src/main/java/frc/team2412/robot/sensors/AprilTagsProcessor.java

```diff
@@ -164,6 +193,35 @@ public void update() {
 		}
 	}
 
+	private void updateSpeakerAngle(PhotonPipelineResult pipelineResult) {
+		int trackingId = (int) trackingIdEntry.getInteger(-1);
+		for (var target : pipelineResult.targets) {
+			if (target.getFiducialId() != trackingId) {
+				continue;
+			}
+			// Camera is rotated so that global left is camera down
+			lastRotatedAngle = Rotation2d.fromDegrees(-target.getPitch());
+			lastRotatedAngleTimestamp = pipelineResult.getTimestampSeconds();
```

Comment:
Add a break; here.  Once you've found the one you want, there is no reason to continue through the rest of the loop.